src/sql/workbench/services/notebook/browser/notebookServiceImpl.ts

```diff
@@ -351,6 +405,8 @@ export class NotebookService extends Disposable implements INotebookService {
 			return undefined;
 		}
 		let uriString = notebookUri.toString();
+		// Format the URI string without query parameters
+		uriString = uriString.split('?')[0];
```

Comment:
Hmm, I'm worried about this change that there's other places the URI is passed around that wouldn't be checking for this. Remind me again - where do you pass in the parameters to the editor/input such that they populate the parameters cell with those values?

Reply:
Parameters would be passed in via the Run Action command which would then append onto the URI based off the part 1 of the logic. As well, as the notebookURIHandler which is able to pass in parameters given the query.

Reply:
Well, I'm pretty sure this is something that's going to come back to haunt us at some point but for now I can't think of any specific examples that would be broken because of this so it's good enough for now. 

Although it might be easier/slightly safer to do 

`let uriString = notebookUri.with({ query: '', fragment: ''}).toString()`

instead to strip out those parts.